draft-ietf-quic-http.md

```diff
@@ -577,17 +569,15 @@ value.
     0                   1                   2                   3
     0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-   |        Identifier (16)        |B|        Length (15)          |
+   |        Identifier (16)        |         Length (16)           |
```

Comment:
That's an interesting discussion.  (I'd agree with Patrick, it's not worth the inconsistency.)  However, *this* Identifier field says which setting the value goes with, so it's definitely necessary.